snapshot/git/gitdb/schemes.go

```diff
@@ -0,0 +1,65 @@
+package gitdb
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/scootdev/scoot/snapshot"
+)
+
+// GitDB uses several different Schemes to identify, upload and download Values.
```

Comment:
I'd get rid of the concept of Value if possible, it adds one too many types. Values are equivalent to different types of snapshots right?